mahjong/index.js

```diff
@@ -486,13 +503,23 @@ module.exports = (clients) => {
 				return;
 			}
 
-			if (text === 'ドラ表示牌') {
+			if (text === '手牌') {
 				if (state.phase !== 'gaming') {
 					perdon();
 					return;
 				}
 
-				postMessage(state.ドラ表示牌s.map((牌) => 牌ToName(牌)).join(' '));
+				// 12r588m239p467s東白白 のように表記
+				const categorized手牌Array = 牌Orders.map((牌Type) => (state.手牌).filter((牌) => get牌Type(牌) === 牌Type));
+				const convertedIntoNumerals手牌Array = categorized手牌Array.map((val) => val.map((牌) => String(数牌ToNumerals(牌))).join(''));
+				const convertedIntoNumerals手牌 = convertedIntoNumerals手牌Array.reduce(
+					(手牌String, categorized手牌String, index) => 手牌String.concat(categorized手牌String, romaji牌Orders[index]), '',
+				);
```

Comment:
[nit] reduceは難しいので使用を避けられる場合は避けると吉です。

今回の場合↓のようにlodash.zipを使用して書くと簡潔に書けると思います。

```suggestion
				const convertedIntoNumerals手牌 =
					zip(convertedIntoNumerals手牌Array, romaji牌Orders)
						.map((牌String, romaji牌Type) => 牌String + romaji牌Type)
						.join('');
```